packages/contracts-bedrock/test/DeploySuperchain.t.sol

```diff
@@ -100,66 +78,52 @@ contract DeploySuperchainOutput_Test is Test {
         // provides precompiles, so we end up with a lot of boilerplate logic to get valid inputs.
         // Hardcoding a concrete set of valid addresses is simpler and still sufficient.
-        DeploySuperchainOutput.Output memory output = DeploySuperchainOutput.Output({
-            superchainProxyAdmin: ProxyAdmin(makeAddr("superchainProxyAdmin")),
-            superchainConfigImpl: SuperchainConfig(makeAddr("superchainConfigImpl")),
-            superchainConfigProxy: SuperchainConfig(makeAddr("superchainConfigProxy")),
-            protocolVersionsImpl: ProtocolVersions(makeAddr("protocolVersionsImpl")),
-            protocolVersionsProxy: ProtocolVersions(makeAddr("protocolVersionsProxy"))
-        });
+        ProxyAdmin superchainProxyAdmin = ProxyAdmin(makeAddr("superchainProxyAdmin"));
+        SuperchainConfig superchainConfigImpl = SuperchainConfig(makeAddr("superchainConfigImpl"));
+        SuperchainConfig superchainConfigProxy = SuperchainConfig(makeAddr("superchainConfigProxy"));
+        ProtocolVersions protocolVersionsImpl = ProtocolVersions(makeAddr("protocolVersionsImpl"));
+        ProtocolVersions protocolVersionsProxy = ProtocolVersions(makeAddr("protocolVersionsProxy"));
 
         // Ensure each address has code, since these are expected to be contracts.
-        vm.etch(address(output.superchainProxyAdmin), hex"01");
-        vm.etch(address(output.superchainConfigImpl), hex"01");
-        vm.etch(address(output.superchainConfigProxy), hex"01");
-        vm.etch(address(output.protocolVersionsImpl), hex"01");
-        vm.etch(address(output.protocolVersionsProxy), hex"01");
+        vm.etch(address(superchainProxyAdmin), hex"01");
+        vm.etch(address(superchainConfigImpl), hex"01");
+        vm.etch(address(superchainConfigProxy), hex"01");
+        vm.etch(address(protocolVersionsImpl), hex"01");
+        vm.etch(address(protocolVersionsProxy), hex"01");
 
         // Set the output data.
-        dso.set(dso.superchainProxyAdmin.selector, address(output.superchainProxyAdmin));
-        dso.set(dso.superchainConfigImpl.selector, address(output.superchainConfigImpl));
-        dso.set(dso.superchainConfigProxy.selector, address(output.superchainConfigProxy));
-        dso.set(dso.protocolVersionsImpl.selector, address(output.protocolVersionsImpl));
-        dso.set(dso.protocolVersionsProxy.selector, address(output.protocolVersionsProxy));
+        dso.set(dso.superchainProxyAdmin.selector, address(superchainProxyAdmin));
+        dso.set(dso.superchainConfigImpl.selector, address(superchainConfigImpl));
+        dso.set(dso.superchainConfigProxy.selector, address(superchainConfigProxy));
+        dso.set(dso.protocolVersionsImpl.selector, address(protocolVersionsImpl));
+        dso.set(dso.protocolVersionsProxy.selector, address(protocolVersionsProxy));
 
         // Compare the test output struct to the getter methods.
```

Comment:
Similar to above, this comment can be updated.

Reply:
Fixed in bd36605f4